protocol-configuration/dependencies.ts

```diff
@@ -178,6 +185,7 @@ const dependencies: DependencyMap = {
       'daiPSM',
```

Comment:
daiPSM should be removed from all dependencies since its deprecated

Reply:
fixed